packages/xsnap-lockdown/package.json

```diff
@@ -42,5 +42,8 @@
     ],
     "timeout": "2m",
     "workerThreads": false
+  },
+  "dependencies": {
```

Comment:
Why does taking a dependency on `@agoric/xsnap-lockdown` also require `@endo/init`?

Reply:
See c3b38d446a3ba98de2d88330ec0d5ea9ca97df15

Alternately, if the modules depending on Endo aren't meant to be exported, we can move them under `scripts` so they don't end up in the NPM distribution. cc @warner

Reply:
The comment (removed in c3b38d4) explains the desired state of the world: the files in `lib` are only meant as build inputs for the generated bundles in `dist`.  All of `lib`'s dependencies should be only build-time dependencies, not runtime dependencies.

I'd suggest removing `"lib"` from `packageJson.files`, as well as moving `xsnap-lockdown/lib` to `xsnap-lockdown/scripts/lib` or somesuch (basically any appropriate directory that allows extraneous imports) and adjusting `scripts/build-bundle.js` accordingly.  Then I think it would make sense to put `"@endo/init"` back into `devDependencies`.

Reply:
Or even better, just remove `"lib"` from `packageJson.files`, remove `packageJson.dependencies`, and don't change the current package layout.

Reply:
I agree with the desired state. I removed the comment because we shouldn't suppress no-extraneous-dependencies.

I tried removing `lib` but `entryPaths` in `src/paths.js` depends on it and that does get exported. Removing no-extraneous-dependencies was some scope creep on this PR so I've reverted those changes to `xnap-lockdown`.

@michaelfig ready to approve? I'll cleanup the commits before merge.